Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:6
+FROM debian:latest
```

Comment:
OK, I understand that.
Your Dockerfile was just one step on my way, so I thought I'd provide a quick-fix. If you want to maintain quality, you are free to reject that fix, but I can't spend more time debugging and improving. 
In the meantime I'd suggest to pin it to `debian:8`.